framework/src/mysql.js

```diff
@@ -368,6 +370,27 @@ const getTableInstance = async (tableName, tableConfig, connEndpoint = CONN_ENDP
 		return query;
 	};
 
+	const update = async (params, trx) => {
+		let isDefaultTrx = false;
+		if (!trx) {
+			trx = await createDefaultTransaction(knex);
+			isDefaultTrx = true;
+		}
+
+		const { where, updates } = params;
+		const query = queryBuilder({ ...where }, tableConfig.primaryKey, trx).update({ ...updates });
```

Comment:
would it be more capsulated if we move the `update({ ...updates })` part inside the `queryBuilder` method?

Reply:
Let's keep it as is. queryBuilder is supposed to be used for all the methods and it's better to keep only the DQL methods within it. Let's avoid adding DMLs within the queryBuilder.